.travis.yml

```diff
@@ -31,7 +31,7 @@ env:
   - PLATFORM=imx8x
   - PLATFORM=imx8m
 
-script: ./scripts/docker-run.sh ./scripts/xtensa-build-all.sh -j $PLATFORM
+script: ./scripts/docker-run.sh ./scripts/xtensa-build-all.sh $PLATFORM
```

Comment:
Sorry, not following the commit message. Why do we not want the -j option here ?

Reply:
Because `-j` option has to provide the number of cores to use. By default, with or without the option and option's value all the cores are used. So this option is not adding any value here. And, with my update, `getopts` enforces that `-j` requires a value with it.

Hence, removing it makes use of all cores like earlier, not altering the CI.